### ISSUE TITLE
Unexpected infinite scrolling and run out of memory with "nativeAndroid" picker

### ISSUE THREAD

**Issue by fluoxe7ine** [henninghall/react-native-date-picker#264]
**Describe the bug**
Wheel starts to scroll infinitely after trying to select date before **minimumDate** or after **maximumDate**
Video process - https://streamable.com/vf3drv

**Expected behavior**
Wheel scrolls to first valid date

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
  <DatePicker
    mode           = {mode}
    date           = {stateValue}
    minimumDate    = {minimumDate}
    maximumDate    = {maximumDate}
    androidVariant = 'nativeAndroid' // if iosClone provided, everything works correctly
    onDateChange   = {this.handleChange}
    locale         = {locale}
/>

```

**Smartphone (please complete the following information):**
 - OS: 
Redmi Note 8 Pro
MIUI version - MIUI Global 11.0.5 Stable 11.0.5.0
Android version - 10 QP1A.190711.020
 - React Native version 0.61.5
 - react-native-date-picker version 3.2.5

**Comment by joaonunogomes** [comment]
Hi! Same here. Have you already find a solution for this issue?

**Comment by fluoxe7ine** [comment]
@joaonunogomes Unfortunately, no ;( Temporary switched to ios version

**Comment by henninghall** [comment]
Fix released in 3.2.8
